test/minter-suite-minters/SetPrice/MinterSetPriceERC20/MinterSetPriceERC20V2.test.ts

```diff
@@ -132,6 +132,77 @@ describe("MinterSetPriceERC20V2_V3Core", async function () {
     MinterSetPriceV2_Common();
   });
 
+  describe("updatePricePerTokenInWei", async function () {
+    it("does not allow price update to be zero", async function () {
+      // does allow artist
+      await expectRevert(
+        this.minter
+          .connect(this.accounts.artist)
+          .updatePricePerTokenInWei(this.projectZero, 0),
+        "Price may not be 0"
+      );
+    });
+  });
+
+  describe("purchase", async function () {
+    it("requires sufficient ERC20 token approval", async function () {
+      // artist changes to Mock ERC20 token
+      await this.minter
+        .connect(this.accounts.artist)
+        .updateProjectCurrencyInfo(
+          this.projectZero,
+          "MOCK",
+          this.ERC20Mock.address
+        );
+      // approve contract and able to mint with Mock token, but insufficient qty approved
+      await this.ERC20Mock.connect(this.accounts.user).approve(
+        this.minter.address,
+        this.pricePerTokenInWei.sub(1)
+      );
+      await expectRevert(
+        this.minter.connect(this.accounts.user).purchase(this.projectZero),
+        "Insufficient Funds Approved for TX"
+      );
+    });
+
+    it("handles ERC20 splits when platform and artist have zero revenues", async function () {
+      // artist changes to Mock ERC20 token
+      await this.minter
+        .connect(this.accounts.artist)
+        .updateProjectCurrencyInfo(
+          this.projectZero,
+          "MOCK",
+          this.ERC20Mock.address
+        );
+      // approve contract and able to mint with Mock token
+      await this.ERC20Mock.connect(this.accounts.user).approve(
+        this.minter.address,
+        this.pricePerTokenInWei
+      );
+      // update platform to zero percent
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .updateArtblocksPrimarySalesPercentage(0);
+      // update artist primary split to zero
+      const proposedAddressesAndSplits = [
+        this.projectZero,
+        this.accounts.artist.address,
+        this.accounts.additional.address,
+        100,
+        this.accounts.additional2.address,
+        50,
```

Comment:
Nit: these two constant values could be helpful to add smol dev comments for.

Reply:
updated 🙏